From 974f6cb106a4e56e3eed3528e0511f756b8da3cd Mon Sep 17 00:00:00 2001
From: Drew Moseley <drew_moseley@mentor.com>
Date: Sat, 5 Jul 2014 18:58:39 -0400
Subject: [PATCH 09/13] init-install-efi.sh: Add a second prompt to install

This also makes it clear that the user will be overwriting
the entire disk.

Signed-off-by: Drew Moseley <drew_moseley@mentor.com>
---
 meta/recipes-core/initrdscripts/files/init-install-efi.sh | 14 ++++++++++++--
 1 file changed, 12 insertions(+), 2 deletions(-)

Upstream-Status: Inappropriate [configuration]

diff --git a/meta/recipes-core/initrdscripts/files/init-install-efi.sh b/meta/recipes-core/initrdscripts/files/init-install-efi.sh
index 2442d5f..84b92b2 100644
--- init-install-efi.sh
+++ init-install-efi.sh
@@ -61,8 +61,18 @@ for hdname in $hdnamelist; do
 	echo "Please answer y or n"
     done
     if [ "$answer" = "y" ]; then
-	TARGET_DEVICE_NAME=$hdname
-	break
+        while true; do
+	    echo "This will overwrite all data on $hdname"
+	    echo -n "Are you sure? [y/n] "
+	    read answer
+	    if [ "$answer" = "y" ]; then
+	        TARGET_DEVICE_NAME=$hdname
+	        break
+            elif [ "$answer" = "n" ]; then
+	        break
+	    fi
+	    echo "Please answer y or n"
+        done
     fi
 
 done
-- 
1.9.1

